spec/jobs/reports/combined_invoice_supplement_report_v2_spec.rb

```diff
@@ -67,16 +64,21 @@
           # 1 new unique user in month 1 at IAA 1 sp @ IAL 1
           7.times do
             create_sp_return_log(
-              user: user1, issuer: iaa1_sp.issuer, ial: 1,
-              returned_at: inside_iaa1
+              user: user1,
+              issuer: iaa1_sp.issuer,
+              ial: 1,
+              returned_at: inside_iaa1,
```

Comment:
Would the same date work for this test? if the ial1 authentication occurred a few days before idv proofing and then subsequent authentication @ ial2, we would want to make sure that 1st ial1 authentication doesn't count right? Am I understanding the "step-up" logic correctly? Or does it only matter in the aggregate which I believe you are account for with the unique users change from 3 to 2?